src/app/dto/AdminDTO/admin.audit.with.pages.DTO.ts

```diff
@@ -0,0 +1,12 @@
+import {AdminAudit} from "./admin.audit.DTO";
+
+export class AdminAuditWithPagesDTO {
```

Comment:
You should have prepared a pull request it only one screen changed to pagination not to make too long for review.